udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDatasetsTable/AdminDatasetsTable.vue

```diff
@@ -0,0 +1,216 @@
+          @sort="(direction) => updateSort('followers', direction)"
+          :sorted="sorted('followers')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-star-s-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Favorites') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="dataset in datasets" :key="dataset.id">
+        <AdminTableTd>
+          <AdminContentWithTooltip>
+            <a class="fr-link" :href="getDatasetLinkToAdmin(dataset)">
+              {{ dataset.title }}
+            </a>
+          </AdminContentWithTooltip>
+        </AdminTableTd>
+        <AdminTableTd>
+          <AdminBadge :type="getStatusType(dataset)">{{ getStatusLabel(dataset) }}</AdminBadge>
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ formatDate(dataset.created_at) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ formatDate(dataset.last_modified) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ getFilesCount(dataset) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          <QualityScoreTooltip :dataset="dataset"/>
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.discussions) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.views) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.resources_downloads ?? 0) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.reuses) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.followers) }}
+        </AdminTableTd>
+      </tr>
+    </tbody>
+  </AdminTable>
+</template>
+<script setup lang="ts">
+import { formatDate } from "@datagouv/components";
+import type { Dataset, DatasetV2 } from '@datagouv/components';
+import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import AdminBadge from "../../AdminBadge/AdminBadge.vue";
+import AdminContentWithTooltip from "../../AdminContentWithTooltip/AdminContentWithTooltip.vue";
+import AdminTable from "../Table/AdminTable.vue";
+import AdminTableTd from "../Table/AdminTableTd.vue";
+import AdminTableTh from "../Table/AdminTableTh.vue";
+import Tooltip from '../../Tooltip/Tooltip.vue';
+import QualityScoreTooltip from "../../dataset/QualityScore/QualityScoreTooltip/QualityScoreTooltip.vue";
+import { admin_root } from '../../../config';
+import { summarize } from '../../../helpers';
+import type { AdminBadgeState, DatasetSortedBy, SortDirection } from '../../../types';
+
+const emit = defineEmits<{
+  (event: 'sort', sort: string): void
+}>();
+
+defineProps<{
+  datasets: Array<Dataset | DatasetV2>
+}>();
+
+const { t } = useI18n();
+
+const sort = ref<SortDirection>('desc');
```

Comment:
```suggestion
const sortDirection = ref<SortDirection>('desc');
```
I think the name `sort` could be ambiguous?